apps/meteor/app/apps/server/marketplace/appInstall.ts

```diff
@@ -0,0 +1,50 @@
+import type { IAppInfo } from '@rocket.chat/apps-engine/definition/metadata';
+
+import { getWorkspaceAccessToken } from '../../../cloud/server';
+import { settings } from '../../../settings/server';
+import { Info } from '../../../utils/server';
+
+export type installAction = 'install' | 'update' | 'uninstall';
+
+export async function notifyAppInstall(marketplaceBaseUrl: string, action: installAction, appInfo: IAppInfo): Promise<void> {
+	let headers = {
+		Authorization: '',
+	};
+
+	try {
+		const token = await getWorkspaceAccessToken();
+		headers = {
+			Authorization: `Bearer ${token}`,
```

Comment:
@graywolf336 if we got an error getting the token, I wonder if we should continue. 🤔 The siteUrl makes sense, as we can send it empty, but for the token, I don't think so 🤔